src/lib/components/carrousel.svelte

```diff
@@ -58,14 +57,14 @@
   .carrousel {
     position: relative;
     width: 100%;
-    aspect-ratio: 16/7;
+    aspect-ratio: 21/7;
```

Comment:
can you put this in a var please, just declare it here if need be --carrousel-aspect-ratio: 21/7